packages/x-date-pickers-pro/src/NextDateRangePicker/describes.NextDateRangePicker.test.tsx

```diff
@@ -0,0 +1,24 @@
+import * as React from 'react';
+import { describeConformance } from '@mui/monorepo/test/utils';
+import { Unstable_NextDateRangePicker as NextDateRangePicker } from '@mui/x-date-pickers-pro/NextDateRangePicker';
+import { wrapPickerMount } from 'test/utils/pickers-utils';
+
+describe('<NextDateRangePicker /> - Describes', () => {
+  describeConformance(<NextDateRangePicker />, () => ({
+    classes: {},
+    muiName: 'MuiNextDateRangePicker',
+    wrapMount: wrapPickerMount,
+    refInstanceof: window.HTMLDivElement,
+    skip: [
+      'componentProp',
+      'componentsProp',
+      'themeDefaultProps',
+      'themeStyleOverrides',
+      'themeVariants',
+      'mergeClassName',
+      'propsSpread',
+      'rootClass',
+      'reactTestRenderer',
+    ],
+  }));
```

Comment:
Is there a reason why not copying the following part from [NextDateRangePicker.test.tsx](https://github.com/mui/mui-x/pull/7135/files#diff-a55031bffcc8e2fd75307dca229fa68208b089c2b4e97fef04471bf1b0ac6a78)?

```js
describeRangeValidation(NextDateRangePicker, () => ({
    render,
    clock,
    componentFamily: 'new-picker',
    views: ['day'],
  }));
```

Reply:
Good spot. Yes, there is. As per Flaviens suggestion, all other tests where respective Mobile and Desktop describe suites. 😉 
But this also raises a good question. Do we need a `describeConformance` test on the wrapping component (i.e. NextDateRangePicker), or should we just put two separate conformance tests in both `Mobile` and `Desktop` versions? 🤔 @flaviendelangle 
From the looks of what the `describeConformance` is doing I'd say it would probable end up being redundant 🤔

Reply:
`describeConformance` would test the theme default props applied on the responsive picker.
That can be worth testing.